src/temporal-logic/normalize_property.cpp

```diff
@@ -119,7 +119,7 @@ exprt normalize_property(exprt expr)
 {
   // top-level only
   if(expr.id() == ID_sva_cover)
-    expr = G_exprt{not_exprt{to_sva_cover_expr(expr).op()}};
+    expr = sva_always_exprt{not_exprt{to_sva_cover_expr(expr).op()}};
```

Comment:
I think we were missing the corresponding comment in `normalize_property.h` -- would you mind adding that one?